src/dumbo/src/tcp/endpoint.rs

```diff
@@ -79,13 +79,13 @@ pub struct Endpoint {
 // is the only option).
 
 impl Endpoint {
+    /// Asserts that RCV_BUF_MAX_SIZE <= MAX_WINDOW_SIZE
```

Comment:
Could we instead write a complete doc comment here with a `## Panics:` section that mentions this simplifying assumption?